blocksync/pool.go

```diff
@@ -602,7 +603,7 @@ OUTER_LOOP:
 			}
 			peer = bpr.pool.pickIncrAvailablePeer(bpr.height)
 			if peer == nil {
-				// log.Info("No peers available", "height", height)
+				bpr.Logger.Debug("No peers available", "height", bpr.height)
```

Comment:
```suggestion
				bpr.Logger.Debug("No peers currently available; will retry shortly", "height", bpr.height)
```
nit: something a little more descriptive